packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -76,3 +81,29 @@ Attribute groups can be imported by their ARN via the `AttributeGroup.fromAttrib
 const importedAttributeGroup = appreg.AttributeGroup.fromAttributeGroupArn(this, 'MyImportedAttrGroup',
   'arn:aws:servicecatalog:us-east-1:012345678910:/attribute-groups/0aqmvxvgmry0ecc4mjhwypun6i');
 ```
+
+## Associations
+
+You associate your appregistry application with attribute groups and resources.
+Resources are Cloudformation stacks that you can associate to application to group relevant
+stacks together to enable metadata rich insights into your applications and resources.
+A Cloudformation stack can only be associated with one application.  If a stack is associated
+with multiple applications in your app or is already associated with one cdk will fail at deploy time. 
+
+### Associating application with an attribute group
+
+You can associate an attribute group with an application with the `associateAttributeGroup()` API:
+
+```ts basic-constructs
+application.associateAttributeGroup(attributeGroup);
+```
+
+### Associating application with a Stack
+
+You can associate a stack with an application with the `associateStack()` API:
+
+```ts basic-constructs
+const myStack = new cdk.Stack(this, 'MyStack');
```

Comment:
`this` should be `app`, no?

Reply:
It doesn't have to be I think, but probably better for end example to be like that so will change.